src/ast/mod.rs

```diff
@@ -2912,24 +2866,25 @@ impl fmt::Display for Statement {
                 }
                 Ok(())
             }
-            Statement::Insert {
-                or,
-                ignore,
-                into,
-                table_name,
-                table_alias,
-                overwrite,
-                partitioned,
-                columns,
-                after_columns,
-                source,
-                table,
-                on,
-                returning,
-                replace_into,
-                priority,
-                insert_alias,
-            } => {
+            Statement::Insert(insert) => {
+                let Insert {
```

Comment:
I don't think it matters, but just FYI you can do the same thing in the single let statement

```suggestion
            Statement::Insert(Insert {
                ..
             }) => {
```

Reply:
Thanks for your comment! I'll remember this and update this snippet when I have some time. You're correct. I just follow the intuitive "refactor" :D